streams/src/main/java/org/apache/kafka/streams/processor/internals/AssignedStandbyTasks.java

```diff
@@ -16,12 +16,42 @@
  */
 package org.apache.kafka.streams.processor.internals;
 
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import org.apache.kafka.common.TopicPartition;
 import org.apache.kafka.common.utils.LogContext;
+import org.apache.kafka.streams.processor.TaskId;
 
 class AssignedStandbyTasks extends AssignedTasks<StandbyTask> {
 
     AssignedStandbyTasks(final LogContext logContext) {
         super(logContext, "standby task");
     }
 
+    /**
+     * Closes tasks that have been reassigned at the end of a rebalance.
+     *
+     * @param revoked the tasks which are no longer owned
```

Comment:
`reassignedTasks`? Also IMHO, these could be simplified as `standbyTasks` and `changelogs`